pallets/pallet-dip-provider/src/lib.rs

```diff
@@ -85,6 +85,11 @@ pub mod pallet {
 	/// double map. Its first key is the `Identifier` of subjects, while the
 	/// second key is the commitment version. The values are identity
 	/// commitments.
+	// SAFETY of Twox64Concat:
```

Comment:
We can still change this, and simply disregard whatever will be written on Peregrine until then, what do you think? I think is's a more sane approach for the pallet to ensure safety here.